### PR TITLE
requirements: Add direct dependencies to Django and celery (fixes #258)

### LINKED ISSUE #258: setup.py does not pull in Django or Celery as direct runtime dependencies
Hi!

I noticed that `setup.py` uses `requirements/*.txt` for dependencies, but none of the ones used during `pip install django-celery-beat` pull in Django and/or Celery, directly. (Django is pulled in indirectly, though). Is that a conscious decision? If not, can they be added?

Best, Sebastian

### PR DESCRIPTION
**Pull request by hartwork**
To address #258

### CODE REVIEW

File: requirements/default.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
+celery
+Django>=1.11.17
```
Comment (liquidpele) [inline requirements/default.txt]: You can't add them to default, or the tests will all fail.  The tests must install a particular version depending on what variables the test is running at the time, and pip complains if you mention the same package twice.  

What you'll probably want to do is update the setup.py file to add just "celery" or "celery<5.0.0" to the install requirements for python3 and python2 respectively.

Comment (hartwork) [reply requirements/default.txt]: Alright, second try :smile:

**Comment by hartwork** [comment]
> This won’t install celery 5.x with python 2 will it? 5.x is what drops py2 support.

Not as long as celery itself gets things right, I believe.

**Comment by hartwork** [comment]
i.e. if they use `python-requires`:
https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

**Comment by hartwork** [comment]
Conflicts are fixed now

**Comment by liquidpele** [comment]
Cool, looks good to me then.

**Pull request merged**